### PR TITLE
Fix test_id:7182

### PR DESCRIPTION
**Pull request by Barakmor1**
We override DeveloperConfiguration in test_id:7182
Instead we should just set DeveloperConfiguration.DiskVerification

This issue cause failure when PSA is enabled
in the cluster becuase we disable PSA featureGate
when overriding DeveloperConfiguration

Signed-off-by: bmordeha <bmodeha@redhat.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/container_disk_test.go [hunk 1]
```diff
@@ -138,10 +138,8 @@ var _ = Describe("[rfe_id:588][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 			It("[test_id:7182]disk verification should fail when the memory limit is too low", func() {
 				By("Reducing the diskVerificaton memory usage limit")
 				kv := util.GetCurrentKv(virtClient)
-				kv.Spec.Configuration.DeveloperConfiguration = &v1.DeveloperConfiguration{
-					DiskVerification: &v1.DiskVerification{
-						MemoryLimit: resource.NewScaledQuantity(42, resource.Kilo),
-					},
+				kv.Spec.Configuration.DeveloperConfiguration.DiskVerification = &v1.DiskVerification{
```
Comment (xpivarc) [inline tests/container_disk_test.go:141]: Do we need to handle a case when `DeveloperConfiguration` could be `nil`?

**Review by xpivarc — COMMENTED** [review]
/approve 
One question.

**Review by fossedihelm — COMMENTED** [review]
Thanks
/lgtm
Let's leave time to respond to lubo's comment
/hold

File: tests/container_disk_test.go (hunk 1 above)
Comment (acardace) [reply tests/container_disk_test.go:141]: Yes we have to.

**Comment by Barakmor1** [comment]
/unhold
we set `DeveloperConfiguration` in :
https://github.com/kubevirt/kubevirt/blob/140b5805dd0054b2385bd4094046794e401f4455/tests/testsuite/kubevirtresource.go#L50

Before running the tests (we also save the old config and restore it afterwards)

So `DeveloperConfiguration` shouldn't be nil, also we check it in here:
https://github.com/kubevirt/kubevirt/blob/140b5805dd0054b2385bd4094046794e401f4455/tests/testsuite/kubevirtresource.go#L76

**Pull request merged**